docs/reference/modules/cluster/misc.asciidoc

```diff
@@ -22,6 +22,27 @@ user with access to the <<cluster-update-settings,cluster-update-settings>>
 API can make the cluster read-write again.
 
 
+[[user-defined-data]]
+==== User Defined Cluster Metadata
+
+User-defined metadata can be stored and retrieved using the Cluster Settings API.
+This can be used to store arbitrary, infrequently-changing data about the cluster
+without the need to create an index to store it. This data may be stored using
+any key prefixed with `cluster.metadata.`.  For example, to store the email
+address of the administrator of a cluster under the key `cluster.metadata.administrator`,
+issue this request:
+
```

Comment:
just some questions for the user's perspective: 

> This data may be stored using

may be? or must be?

Also, could be worth mentioning that settings can be both persistent and transient

Reply:
I think most (all?) settings can be set either/both `persistent` or `transient` - at least, that's how I read the [Cluster Settings API docs](https://www.elastic.co/guide/en/elasticsearch/reference/current/cluster-update-settings.html), so I don't think it's critical to mention that here.